Dockerfile

```diff
@@ -13,6 +17,8 @@ RUN apt-get update \
     && apt-get autoremove \
     && rm -rf /var/lib/apt/lists/*
 
+ENV PYTHON /usr/bin/python3
```

Comment:
Could you potentially resolve the warning made by Github Actions?
![image](https://github.com/user-attachments/assets/030dfc9b-c87a-4b1b-8209-4a1d6ecfd814)

Reply:
Done